closed/openssl-conf/semeru.conf

```diff
@@ -0,0 +1,39 @@
+# ===========================================================================
+# (c) Copyright IBM Corp. 2025, 2025 All Rights Reserved
+# ===========================================================================
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# IBM designates this particular file as subject to the "Classpath" exception
+# as provided by IBM in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+# ===========================================================================
+
+my %targets = (
+	"semeru"                    => { shlib_variant => "-semeru", template => 1 },
+	"semeru-linux"              => {
+		inherit_from            => [ "semeru" ],
+		shared_extension        => ".so",
+		shared_ldflag           => add("-Wl,-rpath,'\$\$'ORIGIN"),
+		template                => 1
+	},
+	"semeru-ap64"               => { inherit_from => [ "aix64-cc" ] },
+	"semeru-ap64-clang"         => { inherit_from => [ "aix64-clang" ] },
```

Comment:
Suggest we can reduce the amount of whitespace before `=>`; it's also misleading having things line up at different levels. A blank line separating the templates from the targets would be welcome:
```
	"semeru"            => { shlib_variant => "-semeru", template => 1 },
	"semeru-linux"      => {
		inherit_from        => [ "semeru" ],
		shared_extension    => ".so",
		shared_ldflag       => add("-Wl,-rpath,'\$\$'ORIGIN"),
		template            => 1
	},

	"semeru-ap64"       => { inherit_from => [ "aix64-cc" ] },
	"semeru-ap64-clang" => { inherit_from => [ "aix64-clang" ] },
```